scripts/deblur

```diff
@@ -54,5 +61,164 @@ def deblur_seqs(seqs_fp, read_error, mean_error, error_dist, indel_prob,
         for s in seqs:
             f.write(s.to_fasta())
 
+
+# TRIM LENGTH COMMAND
+@deblur_cmds.command()
+@click.argument('seqs_fp', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True,
+                                file_okay=True))
+@click.argument('output_fp', required=True,
+                type=click.Path(resolve_path=True, readable=True, exists=True,
+                                file_okay=True))
+@click.option('--trim-length', '-t', required=False, type=int, default=100,
+              help="Sequence trim length")
+def trim(seqs_fp, output_fp, trim_len):
+    """Step 1: trim FASTA sequences"""
+    seqs = trim_seqs(seqs_fp, trim_len)
```

Comment:
you're right, for trimming sequences, the file handler will be better .. however for multiple seq alignment / chimera removal, we'll need the actual file